ngraph/test/runtime/interpreter/evaluates_map.cpp

```diff
@@ -2245,6 +2246,60 @@ namespace
         return true;
     }
 
+    template <element::Type_t ET>
+    bool evaluate(const shared_ptr<op::v6::ExperimentalDetectronGenerateProposalsSingleImage>& op,
+                  const HostTensorVector& outputs,
+                  const HostTensorVector& inputs)
+    {
+        const auto attrs = op->get_attrs();
+        size_t post_nms_count = static_cast<size_t>(attrs.post_nms_count);
```

Comment:
Why do you cast `int64_t` to `size_t`? What if `attrs.post_nms_count < 0`?

Reply:
We cast `int64_t` to `size_t`, because the value of `post_nms_count` is a component of output shapes. Next, `attrs.post_nms_count`, according [the operation specification](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/detection/ExperimentalDetectronGenerateProposalsSingleImage_6.md) cannot be a negative number.

Reply:
`int64_t` doesn't prohibit negative values. Please add an additional check or use `uint64_t`

Reply:
Added check.